projects/isolate_proto/src/isolate_proto/controller.proto

```diff
@@ -165,7 +164,7 @@ message RegisterApplicationRequest {
     // If application has alias: auth mode to use
     optional ApplicationAuthMode auth_mode = 6;
     // Max concurrency in gateway
-    optional int32 max_concurrency = 7;
+    optional int32 max_concurrency = 7 [deprecated = true];
```

Comment:
how does this effect our logic in the server? does old versions of fal cli continue to work because of this?

Reply:
yes, and we handle it explicitly in the controller